api/src/utils/apply-query.ts

```diff
@@ -281,6 +283,37 @@ export function applySort(
 			column[0] = column[0]!.substring(1);
 		}
 
+		// Is the column name one of the aggregate functions used in the query if there is any?
+		if (Object.keys(aggregate ?? {}).includes(column[0]!)) {
+			// If so, return the column name without the order prefix
```

Comment:
Is this comment still correct? 🤔 it mentions returning but doesnt seem linked to any `return` statement

Reply:
The return statements are the ones in this entire if statement.

Reply:
Alright then question 2 isnt the "order prefix" handled above this at line 282-284 outside of the if statement 🤔

Reply:
Not necessarily because the column name would be `sum->id` or whatever the respective column is, not just `sum`. CountAll has no additional column and same with `count of *` so those need to return just the `countAll` or `count` and then that leaves the rest of them as `func->column`. If you go down to applyAggregate you can see the column names it generates.